app/models/community.rb

```diff
@@ -33,6 +33,7 @@ class Community < ApplicationRecord
     {key: :time_zone, type: :time_zone, required: true, default: "UTC"},
     {key: :default_landing_page, type: :enum, options: %w[meals directory reservations wiki],
      default: "directory", required: true},
+    {key: :top_menu_customizations, type: :text},
```

Comment:
For consistency with the rest of the code I think this should be called `main_nav_customizations`.